src/main.c

```diff
@@ -443,6 +446,39 @@ static int poe_set_port_power_up_mode(struct mcu *mcu, uint8_t port[4],
 	return poet_cmd_4_port(mcu, PORT_SET_POE_MODE, port, mode);
 }
 
+/* 0x49 - Get port led map */
+/*static int poe_port_led_map(struct mcu *mcu, uint8_t port_offset)
+{
+	uint8_t cmd[] = { PORT_GET_LED_MAP, 0x00, port_offset };
+
+	return mcu_queue_cmd(mcu, cmd, sizeof(cmd));
+}*/
```

Comment:
Please don't add dead or commented out code. There is a debug packet where you can put the LED mapping, so you can use the get command.